src/azure-cli/azure/cli/command_modules/cosmosdb/commands.py

```diff
@@ -304,3 +340,29 @@ def load_command_table(self, _):
         g.show_command('show', 'get_sql_role_assignment')
         g.command('delete', 'begin_delete_sql_role_assignment', confirmation=True, supports_no_wait=True)
         g.wait_command('wait', 'get_sql_role_assignment')
+
+    with self.command_group('cosmosdb restorable-database-account', cosmosdb_restorable_database_accounts_sdk, client_factory=cf_restorable_database_accounts) as g:
+        g.show_command('show', 'get_by_location')
+        g.custom_command('list', 'cli_cosmosdb_restorable_database_account_list')
+
+    with self.command_group('cosmosdb sql restorable-database', cosmosdb_sql_restorable_database_sdk, client_factory=cf_restorable_sql_databases) as g:
+        g.command('list', 'list')
+
+    with self.command_group('cosmosdb sql restorable-container', cosmosdb_sql_restorable_container_sdk, client_factory=cf_restorable_sql_containers) as g:
+        g.command('list', 'list')
+
+    with self.command_group('cosmosdb sql restorable-resource', cosmosdb_sql_restorable_resources_sdk, client_factory=cf_restorable_sql_resources) as g:
+        g.command('list', 'list')
+
+    with self.command_group('cosmosdb mongodb restorable-database', cosmosdb_mongodb_restorable_database_sdk, client_factory=cf_restorable_mongodb_databases) as g:
+        g.command('list', 'list')
+
+    with self.command_group('cosmosdb mongodb restorable-collection', cosmosdb_mongodb_restorable_collection_sdk, client_factory=cf_restorable_mongodb_collections) as g:
+        g.command('list', 'list')
+
+    with self.command_group('cosmosdb mongodb restorable-resource', cosmosdb_mongodb_restorable_resources_sdk, client_factory=cf_restorable_mongodb_resources) as g:
+        g.command('list', 'list')
```

Comment:
May I know will there be more commands under each command group or just `list` command?

Generally, we don't recommend create command group with single command. Better to rename as `cosmosdb mongodb list-restoreable-collection`, `cosmosdb mongodb list-restoreable-resource`...

Reply:
We'll have more commands in that group.